Jenkinsfile_CNP

```diff
@@ -3,7 +3,7 @@
 @Library("Infrastructure")
 
 def type = "nodejs"
-def product = "ia"
+def product = "wa"
 def component = "wa-proto-frontend"
```

Comment:
I think this way your application will be: wa-wa-proto-frontend

Reply:
good catch! I've just corrected it. Thanks

Reply:
@luigibk ,  could you review, please?

Reply:
Almost there:

charts/ia-wa-proto-frontend/Chart.yaml directory name